roles/installer/tasks/resources_configuration.yml

```diff
@@ -77,6 +77,7 @@
     apply: yes
     definition: "{{ lookup('template', 'deployment.yaml.j2') }}"
     wait: yes
+    wait_timeout: 600
```

Comment:
Please remove this. I feel like it's more likely to cause problems than help anything.

Reply:
Default is 120s:
https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html#parameter-wait_timeout
Initial migration of AWX Database takes longer than 120s, therefore following tasks would be run without AWX pods being ready.
I cannot guess how long it takes in individual environments, as it depends on the circumstances such as throughput, IO and so on.